tests/workflows/pydantic_array_serialization/tests/test_serialization.py

```diff
@@ -0,0 +1,75 @@
+"""
+Test serialization of an array of custom Pydantic models in workflow inputs.
+
+This test verifies that the SDK correctly serializes arrays of Pydantic models
+when they are used as code_inputs in a CodeExecutionNode.
+"""
+
+from deepdiff import DeepDiff
+
+from vellum_ee.workflows.display.workflows.get_vellum_workflow_display_class import get_workflow_display
```

Comment:
for serialization tests, move it to `base_workflow_display` and define the workflow inline